packages/console/src/components/Guide/GuideCardGroup/index.tsx

```diff
@@ -20,6 +23,9 @@ function GuideCardGroup(
   { className, categoryName, guides, hasCardBorder, hasCardButton, onClickGuide }: Props,
   ref: Ref<HTMLDivElement>
 ) {
+  const { isDevTenant } = useContext(TenantsContext);
```

Comment:
The `isDevTenant` likes be handled in the `FeatureTag` component.

Reply:
Discussed offline and this is needed.